cgeo-contacts/src/cgeo/contacts/ContactsActivity.java

```diff
@@ -127,12 +137,9 @@ private void openContactAndFinish(final int id) {
                         !StringUtils.equalsIgnoreCase(foundName, displayName) ? foundName + " (" + displayName + ")" : foundName));
             }
         } catch (final Exception e) {
-            Log.e(LOG_TAG, "ContactsActivity.getContactId", e);
-        } finally {
-            if (cursor != null) {
-                cursor.close(); // no Closable Cursor below sdk 16
```

Comment:
Haven't checked all, but stumbled upon this line: Why don't you close the cursor any more?

Reply:
His visibility has changed.
Previously it was instantiated outside the try block, now inside it.

From code inspection - "'try finally' replaceable with 'try' with resources inspection":
"Reports try finally statements which can use Java 7 Automatic Resource Management. A quickfix is available to convert the try finally statement into a try with resources statement.
This inspection only reports if the project or module is configured to use a language level of 7.0 or higher."

Reply:
Ah, cool, didn't know that feature - thanks!